src/testUtils.ts

```diff
@@ -377,3 +381,291 @@ export const getReserveBalances = async (invariant: InvariantInstance, poolKey:
   const y = await balanceOf(poolKey.tokenY, reserveY)
   return { x, y }
 }
+
+export const feeGrowthFromFee = async (
+  clamm: CLAMMInstance,
+  liquidity: bigint,
+  fee: bigint
+): Promise<bigint> => {
+  return (
+    await clamm.view.feeGrowthFromFee({ args: { liquidity: { v: liquidity }, fee: { v: fee } } })
+  ).returns.v
+}
+
+export const calculateSqrtPrice = async (
```

Comment:
It is already defined in `math.ts`

Reply:
it uses `view` instead of `tests`, it's much faster in general so whenever I had the access to Invariant or CLAMM instance I preferred to use this one